### PR TITLE
feat(wordpress): Adds docker-compose file for wordpress

### PR DESCRIPTION
**Pull request by mwallert**
This file is basically a copy of one Greg found. I commented out the restart: always blocks because I dislike the containers starting on boot.

### CODE REVIEW

File: docker/wordpress/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  wordpress:
+    depends_on:
+      - db
+    image: wordpress:latest # Change this to be version specific if desired
+    #restart: always - I don't like restart always since it will start on boot, leaving this in as an option. MW - 7/28/18
+    volumes:
+      - ./html:/var/www/html
+    environment:
+      WORDPRESS_DB_HOST: db:3306
+      WORDPRESS_DB_PASSWORD: p4ssw0rd!
```
[the remark this answers is not in the document]
Comment (mwallert) [reply docker/wordpress/docker-compose.yml]: Yeah we could have it read from an env file or something of that nature. Thoughts on this guys?

[the remark this answers is not in the document]
Comment (ryekerjh) [reply docker/wordpress/docker-compose.yml]: Also, if we do this via .env, should we change the sql_passwords to .env vars as well?

[the remark this answers is not in the document]
Comment (zbyte64) [reply docker/wordpress/docker-compose.yml]: IMHO if docker compose is setting the database password then it should specify the database password for the application. `.env` will be overridden by environment variables set it docker-compose.yml though, I wish it were the other way around.

**Comment by mwallert** [comment]
Gonna go ahead and merge this, for WP dev I think we are good here, shoutout to @ggoforth for finding this and @arturoceballos for sharing!

**Pull request merged**